.gitignore

```diff
@@ -23,6 +23,7 @@ packages/
 .pub
 .packages
 .vagrant/
+.vscode/
```

Comment:
I think, this should be added locally on your own machine to the global git ignore file: https://davidwalsh.name/global-gitignore